site/_includes/themes/apache-clean/_navigation.html

```diff
@@ -40,7 +40,6 @@
         <ul class="nav navbar-nav">
           {% for entry in site.data.navigation.topnav %}
           {% comment %}<!-- exclude Download from navbar on the Home page -->{% endcomment %}
-          {% if page.title != "Home" or entry.title != "Download" %}
```

Comment:
This change lets the "Download" menu dropdown show on all pages, including "Home".